src/Service/Documents.php

```diff
@@ -38,4 +39,63 @@ public function Metadata(int $docId)
 
         throw new InvalidResponseException('Metadata missing in response');
     }
+
+    /**
+     * @throws GetOrganizedClientException
+     * @throws InvalidFilePathException
+     */
+    public function AddToDocumentLibrary(string $filePath, string $caseId, string $fileName = '', array $metaDataOptions = [], bool $overWrite = true, string $listName = 'Dokumenter', string $folderPath = '')
```

Comment:
I suggest renaming `$metadataOptions` to just `$metadata`. And `$overWrite` to `$overwrite`.